docs/content/kv_commands.md

```diff
@@ -58,18 +84,19 @@ Outputs a list of all KV namespaces associated with your account id.
 $ wrangler kv:namespace list
 🌀  Retrieving namespaces
 ✨  Success:
-+------------------+----------------------------------+
-| TITLE            | ID                               |
-+------------------+----------------------------------+
-| new kv namespace | f7b02e7fc70443149ac906dd81ec1791 |
-+------------------+----------------------------------+
++---------------+----------------------------------+
+| TITLE         | ID                               |
++---------------+----------------------------------+
+| NEW_NAMESPACE | f7b02e7fc70443149ac906dd81ec1791 |
++---------------+----------------------------------+
 ```
 
 ## `kv:key`
 
 ### `put`
 
 Writes a single key/value pair to the given namespace. Optional params include 
+1. `--env`: The environment argument.
```

Comment:
expand on this as if the user has no idea what environments are. perhaps go so far as to link to environments doc page.